db/migrate/20220106122727_create_reminders.rb

```diff
@@ -0,0 +1,9 @@
+class CreateReminders < ActiveRecord::Migration[6.0]
+  def change
+    create_table :reminders do |t|
+      t.datetime :due_at, null: false
+      t.string :state_class_type, null: false
+      t.jsonb :issue_status_changed
```

Comment:
A esto quizás no lo pondría tan específico, así no agregamos una por cada regla nueva. Pondría algo como:
```suggestion
      t.jsonb :transition_rules
```
Donde dentro definimos un key para cada caso.

Reply:
esto no te entendi franco, lo cambio y mañana te pregunto